README.md

```diff
@@ -25,6 +25,17 @@ treefmt-nix.lib.mkWrapper nixpkgs {
 }
 ```
 
+### `flake-parts`
+
+This flake exposes a [flake-parts](https://flake.parts/) module as well. To use it:
+
+1. Add `inputs.treefmt-nix.flakeModule` to the "imports" list of your `flake-parts` call.
+1. Add `treefmt = { .. }` (containing the configuration above) to your "perSystem" section.
+1. Add `config.treefmt.build.wrapper` to the `buildInputs` of your devShell. This will make the `treefmt` command available in the shell using the specified configuration.
```

Comment:
```suggestion
1. Add `config.treefmt.build.wrapper` to the `nativeBuildInputs` of your devShell. This will make the `treefmt` command available in the shell using the specified configuration.
```